Droneappmerge

rwreinert opened the pull request:
Closes issues #3 #4 #5.

Adds functionality to send commands to Cherry Py directly from the wb app.

rwreinert committed:
Updates the drone kit home button.

Adds script call when pressed.

rwreinert committed:
Optimizes js code for index page.

Adds external js call and updated js file.

rwreinert committed:
Delete index_buttons.js

rwreinert committed:
Add external js script for home page.

rwreinert committed:
Add Py controls to web app.

Adds Py commands to web app, including halt, start, and home.

lwreinert commented on RaspberryPi/WebApp/scripts/home.php:
```diff
@@ -0,0 +1,22 @@
+
+<?php
+
+$url = "http://localhost:9000/home";
```
Can you use a variable (based upon a configuration file entry)instead of localhost (which defaults to localhost)? That would make it easier to move the web app to another device/platform (such as an Android app).

rwreinert replied:
Completed 8/12/2016

lwreinert commented on RaspberryPi/WebApp/scripts/pause_resume.php:
```diff
@@ -11,9 +11,16 @@
 
-echo "Doesn't work yet.";
-exec('');
+$url = "http://localhost:9000/resume";
```
Same comment as above for localhost.

rwreinert replied:
Completed 8/13/2016

lwreinert commented:
See comments in code about making use of  a variable (based upon a configuration file entry)instead of localhost. That should make it easier to move the web app to another device/platform (such as an Android app).

rwreinert committed:
Add config file

Add config file to allow modification of the local host.

rwreinert committed:
Add minor update

Add minor update to code

rwreinert committed:
Complete config file for Py commands

Completes the incorporation of an external config file to adjust the
local host for Cherry Py commands.

lwreinert commented:
:+1:  Looks good
